translations/test/ElementMergeServer.js

```diff
@@ -92,12 +92,12 @@ var buildingsInput =
          <tag k='name' v='building 1' />\
          <tag k='hoot:merge:target' v='yes'/>\
        </way>\
-       <way id='-222332' action='modify' visible='true'>\
-         <nd ref='-222264' />\
-         <nd ref='-222262' />\
-         <nd ref='-222260' />\
-         <nd ref='-222258' />\
-         <nd ref='-222264' />\
+       <way id='222332' action='modify' visible='true'>\
+         <nd ref='222264' />\
+         <nd ref='222262' />\
+         <nd ref='222260' />\
+         <nd ref='222258' />\
+         <nd ref='222264' />\
```

Comment:
* changing these ids to positive, so that a positive id entity being merged into a negative id entity causes the exception show.
* positive into positive is fine and negative into negative is fine.
* nodes do not seem affected by this.